### DISCUSSION TITLE
Should I use `actix-ws` or `actix-ws-ng`?

### DISCUSSION THREAD

**Discussion by DenuxPlays** [actix/actix-extras#433] [Q&A] [answered]
I am a little bit confused on what to use.
actix-ws was last updated 2 years ago but its contained in this repository.
acitx-ws-ng is newer with some extra features but the repository is archived, but in the readme in this repositroy it says to use actix-ws-ng.
What should I use?
What package will be updated when a new update will be released?

**Comment by robjtede** [comment, marked as answer]
You should use `actix-ws`. It will continue to be developed.
